src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/LayoutTestControl/VerticalStackPanelTest.xaml.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Uno.UI.Samples.Controls;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The User Control item template is documented at https://go.microsoft.com/fwlink/?LinkId=234236
+
+namespace UITests.Shared.Windows_UI_Xaml_Controls.LayoutTestControl
+{
+	[SampleControlInfo("LayoutTestControl", nameof(VerticalStackPanelTest))]
+	public sealed partial class VerticalStackPanelTest : UserControl
+	{
+		public VerticalStackPanelTest()
+		{
+			this.InitializeComponent();
+		}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this initialization to '_currentWidth', the compiler will do that for you.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826544)